### PR TITLE
chore: create dev config for Will Haley

### PR DESCRIPTION
**Pull request by williamhaley**
Jira Ticket: [PXP-xxxx](https://ctds-planx.atlassian.net/browse/PXP-xxxx)
- [ ] Remove this line if you've changed the title to (PXP-xxxx): <title>

### CODE REVIEW

File: will.planx-pla.net/mpingram.planx-pla.net/etlMapping.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+mappings:
+  - name: mpingram_etl
```
Comment (emalinowski) [inline will.planx-pla.net/mpingram.planx-pla.net/etlMapping.yaml]: Might want to change this, any ETL jobs will override his indices. If you want to rely on his indices and not run ETL until you get the hang of it just remove this file, or else change the name.

File: will.planx-pla.net/mpingram.planx-pla.net/etlMapping.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      - index: file
+        join_on: case_id
+        props:
+          - name: data_format
+            src: data_format
+            fn: set
+          - name: data_type
+            src: data_type
+            fn: set
+  - name: mpingram_file
```
Comment (emalinowski) [inline will.planx-pla.net/mpingram.planx-pla.net/etlMapping.yaml]: Same here

File: will.planx-pla.net/manifest.json [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+    "public_datasets": true,
+    "netpolicy": "on"
+  },
+  "canary": {
+    "default": 0
+  },
+  "guppy": {
+    "indices": [
+      {
+        "index": "mpingram_etl",
```
Comment (emalinowski) [inline will.planx-pla.net/manifest.json:244]: Change this to match what is in the ETL file, if you change the name.

File: will.planx-pla.net/manifest.json [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+        "type": "case"
+      },
+      {
+        "index": "mpingram_file",
```
Comment (emalinowski) [inline will.planx-pla.net/manifest.json:248]: Same here

File: will.planx-pla.net/manifest.json [hunk 5]
```diff
@@ -0,0 +1,294 @@
...
+        "type": "file"
+      }
+    ],
+    "config_index": "mpingram_array-config",
```
Comment (emalinowski) [inline will.planx-pla.net/manifest.json:252]: Same here

File: will.planx-pla.net/mpingram.planx-pla.net/etlMapping.yaml (hunk 1 above)
Comment (williamhaley) [reply will.planx-pla.net/mpingram.planx-pla.net/etlMapping.yaml]: Thanks! I'll update to my name

File: will.planx-pla.net/portal/gitops.json [hunk 6]
```diff
@@ -0,0 +1,314 @@
...
+        "name": "Experiment"
+      }
+    ],
+    "projectDetails": "boardCounts"
+  },
+  "components": {
+    "appName": "The Michael Commons Portal",
+    "index": {
+      "introduction": {
+        "heading": "Michael Commons",
```
Comment (emalinowski) [inline will.planx-pla.net/portal/gitops.json:37]: It will just be cosmetic, but you can change the references to Michaels commons here.

**Review by paulineribeyre — APPROVED** [review]
you should be fine with this config as long as you're not trying to get the ETL/Guppy/Exploration page to work. I think you can merge it as is for now, and fix what needs to be fixed when you'll need to :p

**Review by emalinowski — APPROVED** [review]

**Pull request merged**